docs/hpo/strategies.md

```diff
@@ -0,0 +1,163 @@
+# Quick Start
+
+Using maggy for Hyperparameter Optimization (HPO) works as follows:
+
+* Define a training function containing the training logic.
+```py
+def training_function(hparams):
```

Comment:
Is it config or hparams?

We should be consistent across all examples.
Honestly, i like hparams, but if you have config that is ok, took. Just pick one.

Reply:
Config is passed to lagom, not to the training function, I think I'll keep it as it is

Reply:
Actually, I'll change the parameters to  'model, train_set, ...'